### PR TITLE
Update default branch: master -> trunk

### PR DESCRIPTION
**Pull request by texasmichelle**
This change is [Reviewable](https://reviewable.io/reviews/compnerd/swift-build/324)

### CODE REVIEW

File: tensorflow.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <manifest project="swift">
   <remote name="github" fetch="git://github.com" />
 
-  <default revision="master" />
+  <default revision="trunk" />
```
Comment (compnerd) [inline tensorflow.xml:5]: This impacts a number of repositories bellow (i.e. anything that does not explicitly specify a revision) so I think that we should change this to `main` and update any of the revisions that explicitly are using `main` to that, and use the explicit name for the other ones.

**Review by compnerd — CHANGES_REQUESTED** [review]

File: .ci/templates/toolchain.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ parameters:
 
   - name: VERSION
     type: string
-    default: 'master'
+    default: 'trunk'
```
Comment (compnerd) [inline .ci/templates/toolchain.yml:28]: Does this not need changes in scripts to ensure that the layout is properly selected?

File: .ci/templates/windows-sdk.yml [hunk 3]
```diff
@@ -38,7 +38,7 @@ parameters:
 
   - name: VERSION
     type: string
-    default: 'master'
+    default: 'trunk'
```
Comment (compnerd) [inline .ci/templates/windows-sdk.yml:41]: Similar

File: docs/Windows.md [hunk 4]
```diff
@@ -15,10 +15,10 @@ To open the Native Tools Command Prompt with administrator rights, navigate to t
 Then execute the following four commands from within the Native Tools Command Prompt:
 
 ```cmd
-curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/ucrt.modulemap" -o "%UniversalCRTSdkDir%\Include\%UCRTVersion%\ucrt\module.modulemap"
-curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/visualc.modulemap" -o "%VCToolsInstallDir%\include\module.modulemap"
-curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/visualc.apinotes" -o "%VCToolsInstallDir%\include\visualc.apinotes"
-curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/winsdk.modulemap" -o "%UniversalCRTSdkDir%\Include\%UCRTVersion%\um\module.modulemap"
+curl -L "https://raw.githubusercontent.com/apple/swift/main/stdlib/public/Platform/ucrt.modulemap" -o "%UniversalCRTSdkDir%\Include\%UCRTVersion%\ucrt\module.modulemap"
```
Comment (compnerd) [inline docs/Windows.md:18]: This can be split out into a separate change I think.

**Review by compnerd — CHANGES_REQUESTED** [review]

File: docs/Windows.md (hunk 4 above)
Comment (texasmichelle) [reply docs/Windows.md:18]: https://github.com/compnerd/swift-build/pull/330

**Pull request closed without merging**